packages/steps/src/utils/template.ts

```diff
@@ -24,6 +25,25 @@ export function interpolateWithOutputs(
   return interpolate(templateString, BUILD_STEP_OUTPUT_EXPRESSION_REGEXP, fn);
 }
 
+/**
+ * Returns the selected status check from the if statement template.
+ * Example: `${ success() }` -> `success`
+ */
+// TODO @szdziedzic: add support for cancelled
```

Comment:
Should we create a Linear task for it already? Maybe replace this with a link to Linear task? There's enough `// TODO`s as it is. 😅

Reply:
Added: https://linear.app/expo/issue/ENG-10700/investigate-adding-canceled-conditional-check-for-generic-job-steps